include/circt/Dialect/FIRRTL/InnerSymbolTable.h

```diff
@@ -182,6 +182,30 @@ class InnerSymbolTable {
   /// A successful walk with no failures returns success.
   static LogicalResult walkSymbols(Operation *op, InnerSymCallbackFn callback);
 
+  /// Get the sym name from the InnerSymPropertiesAttr.
+  static StringAttr getInnerSymName(const InnerSymPropertiesAttr p) {
```

Comment:
If this method stays here, please align this "get inner symbol from a thing" method with the ones above `walkSymbols`, currently called `getInnerSymbol`.  Not adamant about which name but having both is confusing.

That said, hopefully users don't have to deal with `InnerSymProperties` too often?  They're only available via the new ISA iterators (begin/end) and have their own getName method so I'm not sure what this is useful for.

Reply:
I agree this seemed a bit off. I just thought it would be nice to have, but there's no use case. So, got rid of it.